egs2/mini_an4/asr1/pyscripts/feats/compute-cmvn-stats.py

```diff
@@ -0,0 +1 @@
+../../../../../utils/compute-cmvn-stats.py
```

Comment:
do you need this?
the current `path.sh` set a path in `utils`.

Reply:
I didn't decide my mind about which scripts are placed under the `egs2/*asr1/` and which commands should invoked directly using `${PATH}`.

At least, shell scripts should be put under the `egs/` because they can't be used in the other space due to dependencies on `path.sh`, `utils`, etc.